website/docs/developer/deployment-guide/deployment-with-cumulus.md

```diff
@@ -504,6 +498,8 @@ accessed using terraform dot syntax in the format of `module.orca.variable_name`
 | `orca_sqs_staged_recovery_queue_id`                  | The URL ID of the staged-recovery-queue SQS |
 | `orca_sqs_status_update_queue_arn`                   | The ARN of the status-update-queue SQS |
 | `orca_sqs_status_update_queue_id`                    | The URL ID of the status-update-queue SQS |
+| `orca_secretsmanager_arn`                            | The Amazon Resource Name (ARN) of the AWS secretsmanager |                                 
```

Comment:
Put in alphabetical order please. Should appear before sqs variables

Reply:
fixed